core/ops_builtin.rs

```diff
@@ -19,97 +21,102 @@ use std::io::stdout;
 use std::io::Write;
 use std::rc::Rc;
 
-deno_core::extension!(
-  core,
-  ops = [
```

Comment:
`core` is not longer an extension - this has a nice side-effect that it's enforced by the type system that `core` doesn't contribute to `OpState` the way other extensions do.